CHANGELOG.md

```diff
@@ -10,6 +10,56 @@ and this project adheres to [Semantic Versioning].
 ### Added
 
 - API for resolving imported function names into function types. [#318](https://github.com/wasmx/fizzy/pull/318)
+- Validation of stack height at the end of blocks.  [#319](https://github.com/wasmx/fizzy/pull/319) 
+  [#356](https://github.com/wasmx/fizzy/pull/356)
+- Validation of stack height at the end of functions. [#343](https://github.com/wasmx/fizzy/pull/343)
```

Comment:
We could group all of stack height validations together

Reply:
Sounds good, just mention these 4 cases in the description.